src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,40 @@
-import React from 'react';
+import { FC } from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: FC<Props> = ({
+  comment: {
+    email,
+    name,
+    body,
+  },
+}) => {
+  const emailString = `mailto:${email}`;
+
+  return (
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {name}
+        </strong>
+
+        {' by '}
+
```

Comment:
Try to use styles instead of spaces
```suggestion

        <span class="box">by</span>

```